---
name: Tiffany Amariuta
email: tamariutabartell@ucsd.edu
photo: assets/images/tiffany-amariuta.png
website: https://www.amariutalab.org/
domain: A01
title: Precision Genomics with Personalized Genetic Risk Prediction
bio: "Before starting her lab in San Diego, Tiffany earned a B.S. in Biological Engineering at MIT and went on to conduct graduate research with Dr. Soumya Raychaudhuri as part of the Bioinformatics and Integrative Genomics PhD program at Harvard Medical School, where she studied the genetic susceptibility of autoimmune diseases and other polygenic diseases. During graduate school, Tiffany developed machine learning methods to predict the functionality of regulatory variants, which had applications to transcription factor binding prediction, eQTL mapping, heritability enrichment analysis, and trans-ancestry portability of polygenic risk scores. She pursued post-doctoral research studying tissue-mediated genetic effects with Dr. Alkes Price at the Harvard School of Public Health. Now, Tiffany is an Assistant Professor in the Halıcıoğlu Data Science Institute and the Department of Medicine at the University of California San Diego. In her free time, Tiffany enjoys figure skating, hiking, tennis, beach volleyball, and spending time with her dog, Dax."
description: "A polygenic risk score (PRS) is a weighted sum of an individual’s risk alleles across one’s genome for a particular phenotype, i.e. disease or other measurement. The weights are typically the effect sizes of the risk allele, estimated by a genome-wide association study (in the case of complex traits / polygenic diseases) or an eQTL study (in the case of gene expression). PRS have great potential to revolutionize preventive care. In theory, an individual may arrive at the clinic not knowing their genetic susceptibility to a disease, have their DNA sequenced, and learn what is their lifetime risk for the disease. There is a theoretical liability threshold of PRS at which individuals who have a PRS value lower than the threshold will not develop the disease and those with a value higher than the threshold will develop disease. For diseases with a monogenic basis, it has been shown that the same degree of disease risk can be conferred by polygenic risk alone (Khera 2018 Nature Genetics). PRS are generally useful for understanding how predictive genetics is of disease and how disperse the genetic contributions are. PRS is especially useful in understanding genetic liability when individual effects are too small to be easily detected by genome-wide association studies (Purcell 2009 Nature). In this capstone, students will use population genetics and genomics data to assess individual risk for disease outcomes and transcriptomic measurements. Students will learn to work with genotype data from 1000Genomes and genetic association data from genome-wide association studies (GWAS) and transcriptome-wide association studies (TWAS)."
summer: "Summer reading: 
Study that generated data we will use: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3918453/
Polygenic risk scores: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3912837/, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6128408/
Privacy concerns regarding genomic data: https://www.nature.com/articles/s41576-022-00455-y
Genetic effects on gene expression: https://pubmed.ncbi.nlm.nih.gov/32913098/"
oldstudent: https://antonbeliakovucsd.github.io/Capstone/
prerequisites: "Bioinformatics, gene expression (RNA-seq) data, genotyping data, computational biology, genomics"
time: Wednesday 11AM-12PM, In-Person
style: I run the capstone independently, without help from grad students or postdocs in my own lab. I like to meet with everyone simultaneously and lead discussion sections each week about the papers/data we are looking at and analyses students are doing. I am hands on in the sense that the weekly tasks are predetermined rather than being abstract; this changes in the second quarter where students will follow their own research plan (with my input and guidance).
seats: 8
tag: Bio
---